README.md

```diff
@@ -24,6 +24,10 @@ See the [documentation](doc/PodioInputOutput.md) for more information.
 
 Algorithm merging multiple collections of the same type into a single collection.
 
+### EfficiencyFilter
+
+Algorithm that takes a collection and creates a subset collection with a random sample of the elements of the collection.
```

Comment:
```suggestion
Algorithm that takes a collection and creates a subset collection by randomly rejecting elements according to the user defined `Efficiency`.
```

Reply:
I am not sure there is a use case for `Exact = True` but ok, it is there already